src/lang/en.json

```
@@ -891,6 +891,8 @@
     "deleteRemoteBrowserMessage": "Are you sure want to delete this Remote Browser for all monitors?",
     "GrafanaOncallUrl": "Grafana Oncall URL",
     "Browser Screenshot": "Browser Screenshot",
+    "Zookeeper Host": "Zookeeper Host",
+    "Zookeeper Timeout": "Zookeeper Connect Timeout (ms)"
```

typo introduced by a merge

```suggestion
    "Zookeeper Timeout": "Zookeeper Connect Timeout (ms)",
```